.env.tpl

```diff
@@ -19,6 +19,7 @@ AUTH_SECRET=keycloakZaakafhandelcomponentClientSecret
 AUTH_SERVER=http://localhost:8081
 BAG_API_CLIENT_MP_REST_URL=op://Dimpact/ZAC-.env-$APP_ENV/BAG/API_CLIENT_MP_REST_URL
 BAG_API_KEY=op://Dimpact/ZAC-.env-$APP_ENV/BAG/API_KEY
+BRON_ORGANISATIE=123443210
```

Comment:
Maybe use `BRON_ORGANISATIE_BSN` instead? Because that immediately shows that this is a BSN and not just any number. Same with the other one.

Reply:
Changed